### ISSUE TITLE
Thingsboard Get Telemetry API

### ISSUE THREAD

**Issue by lmax25** [thingsboard/thingsboard#3723]
Please tell me how can I run a few API commands from a custom widget !

I have found these commands on Swagger UI REST API.

One of the commands is to get JWT TOKEN :

curl -X POST --header 'Content-Type: application/json' --header 'Accept: application/json' -d '{"username":"tenant@thingsboard.org", "password":"tenant"}' 'http://127.0.0.1:8080/api/auth/login'

The second command is for getting telemetry data for a specified interval from one device :

http://127.0.0.1:8080/api/plugins/telemetry/DEVICE/0f94acf0-0e4b-11eb-bdd9-0507c88fa6e5/values/timeseries?limit=10000000&agg=NONE&useStrictDataTypes=false&keys=key1,key2,key3&startTs=1602460059000&endTs=1603237659000

I don't know how to call the telemetry service from widget.

I think if you call the telemetry service from widget, to get the JWT TOKEN will be not required..

Can you help me, please ?

Thank You !

**Environment**
 * OS:  Win7 SP1 x64
 * ThingsBoard: 3.1
 * Browser: Chrome

**Comment by dhaval2017** [comment]
you are right no need of JWT TOKEN if you call service to get telemetry data
self.ctx.attributeService.getEntityAttributes() 

https://github.com/thingsboard/thingsboard/blob/eb16527e4dc6017038c5f3dc93e04a953b19299a/ui-ngx/src/app/core/http/attribute.service.ts#L35

**Comment by PovitruliaV** [comment]
Hi @lmax25,
Is this issue still relevant?
